### DISCUSSION TITLE
Migrating to new Docker install

### DISCUSSION THREAD

**Discussion by charmarkk** [automatic-ripping-machine/automatic-ripping-machine#755] [Q&A]
I have an older version which was compiled/installed long before the Docker installs were a thing. To upgrade/migrate to the Docker methods, should we just remove all the previous things that were installed manually or via the Ubuntu scripts and reboot? Directories, YAMLs, etc?

ETA or is it worth migrating at all, if it's still working? Short of having to manually change some settings in abcde etc. vs. having the WebUI to do that. Especially having to learn Docker etc.

**Comment by shitwolfymakes** [comment]
If you don't mind waiting, the updated installer script will handle that automagically!

**Reply by charmarkk** [reply]
Excellent news! I know enough of what I'm doing but having a script do it is much better. :) In no rush!

**Reply by charmarkk** [reply]
@shitwolfymakes Any updates on this? Looks like there's been several version updates since my last install.

**Reply by shitwolfymakes** [reply]
That's fallen by the wayside, and due to the architecture changes we're adding with 3.0 the installer script is basically no longer necessary. Might as well back up your database file, wipe the VM and start fresh with the docker installation method.

**Reply by charmarkk** [reply]
This is just installed on an Ubuntu server not a VM, wiping isn't really a viable option (rather just removing all of the old files it sounds like) but I get the idea. I'll keep that in mind, thanks! Maybe finally an excuse to learn Docker's way of doing things. :)
